sdk/openai/openai/package.json

```diff
@@ -61,7 +61,8 @@
       "commonjs"
     ],
     "esmDialects": [
-      "browser"
+      "browser",
+      "react-native"
```

Comment:
Stainless library doesn't support react: checkout the last couple sentences in https://github.com/openai/openai-node